OCR/frontend/src/pages/AnnotateTemplate.tsx

```diff
@@ -34,44 +47,123 @@ const AnnotateTemplate: React.FC = () => {
         const context = canvas.getContext("2d")!;
         canvas.height = viewport.height;
         canvas.width = viewport.width;
-        await page.render({ canvasContext: context, viewport: viewport }).promise;
+        await page.render({ canvasContext: context, viewport: viewport })
+          .promise;
         images.push(canvas.toDataURL());
       }
       canvas.remove();
+      URL.revokeObjectURL(data);
       return images;
-    }
+    };
 
     convertPdfToImages(pdfFile).then((imgs) => {
       setImages(imgs);
-      console.log(imgs)
     });
+  }, [files]);
 
-  }, [files])
+  const renderLabelContent = (category: LabelCategory): JSX.Element => (
+    <ul className="usa-list usa-list--unstyled">
+      {category.items.map((item) => (
+        <li
+          key={item.name}
+          style={{
+            display: "flex",
+            justifyContent: "space-between",
+            alignItems: "center",
+            padding: "8px 0",
+          }}
+        >
+          <div style={{ display: "flex", alignItems: "center" }}>
```

Comment:
can we use the [style guide](https://designsystem.digital.gov/utilities/display/) for all of the class names

Reply:
I have tried to follow the style guide as close as I can please let me know if I missed anything! Happy to pair on it as well.